packages/lit-helpers/src/spread.ts

```diff
@@ -30,9 +30,9 @@ type EventListenerWithOptions = EventListenerOrEventListenerObject &
 export class SpreadPropsDirective extends AsyncDirective {
     host!: EventTarget | object | Element;
     element!: Element;
-    prevData: { [key: string]: unknown } = {};
+    prevData: Record<string, any> = {};
```

Comment:
What's the functional difference between `any` and `unknown` here?

Also, will this cause type issue in the other direction?